cuqi/experimental/mcmc/_gibbs.py

```diff
@@ -255,10 +255,11 @@ def _get_initial_points(self):
         """ Get initial points for each parameter """
         initial_points = {}
         for par_name in self.par_names:
-            if hasattr(self.target.get_density(par_name), 'init_point'):
-                initial_points[par_name] = self.target.get_density(par_name).init_point
```

Comment:
I wonder if we have to make an issue to remove the initial point from distributions in the future. Seems unintuitive to me. (by in the future I mean after removing the old Sampler module)

Reply:
Actually they were not part of the distributions at all. If was this "hack" that one had to add an attribute to an instance of a distribution. So this removes it entirely.

Reply:
Okay I see now, thanks for the explanation :)